app/controllers/users_controller.rb

```diff
@@ -1,3 +1,30 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by(id: params[:id])
+    return if @user
+
+    flash[:danger] = t("user_not_found")
+    redirect_to root_url
```

Comment:
```suggestion
    @user = User.find_by id: params[:id]
    return if @user

    flash[:danger] = t "user_not_found"
    redirect_to root_path
```
dùng `_path` thay vì Url em nhé,. chỗ này dùng đường dẫn tương đối dc rồi